(testing) return half of the orchestrators as untrusted

darkdarkdragon opened the pull request:
**What does this pull request do? Explain your changes. (required)**

PR for fast verification testing in staging

**Specific updates (required)**

<!--- List out all significant updates your code introduces -->

## -

- **How did you test each of these updates (required)**

<!-- A detailed description of how you tested your code changes. Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc. -->

**Does this pull request close any open issues?**

<!-- Fixes # -->

**Screenshots (optional):**

<!-- Drag some screenshots here, if applicable -->

**Checklist:**

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

darkdarkdragon committed:
return half of the orchestrators as untrusted

darkdarkdragon committed:
return half of the orchestrators as untrusted

victorges commented on packages/api/src/parse-cli.ts:
```diff
@@ -242,6 +242,12 @@ export default function parseCli(argv?: string | readonly string[]) {
+          "mark half of the orchestrators returned by /api/region as untrusted. For use in staging!",
+        default: false,
+        type: "boolean",
+      },
```
Maybe we don't even need to merge this and thus don't need the flag. We can just change the image used for staging deployments on `livepeer-infra` to be the one coming from this branch.

darkdarkdragon replied:
I did it initially, but then it was overwritten in today's deploy 😢 
But yah, let's try it again

victorges replied:
Oh I see. We could merge to `master` the change on `livepeer-infra` values/image for the staging deploy. The flag/env setting would also be overwritten if we didn't merge it anyway

victorges replied:
So... I wasn't aware this would be such a long running test. I think we should actually merge this and add the CLI arg only to staging instead as you were planning on doing originally. Otherwise, no new changes are being picked up by staging when we merge to `master`. WDYT?

victorges replied:
For now, I rebased this on top of `master` if you don't mind, which is OK as well for me. Just keep in mind that you might need a `reset` or `pull --rebase` to update to the `origin` version of the branch now

thomshutt replied:
Hey @victorges, I'm picking this up again - is reinstating the flag and passing that in for Staging still the way you want to go?

victorges replied:
Sure! I think in the end this was a long-running test that we wanna keep going apart from other potential tests we want to make. In hindsight it would've been better to just merge this to master but only configure the flag in staging. 

So it makes more sense to pick this up, merge it to `master`, and set the flag only on staging! Let me know when you rebase this and it's ready for a review/merging.

victorges committed:
api: Set some default profiles to streams (#875)

* api: Set some default profiles in case none specified

It's kind of a roadblock for initial developers trying to test
Livepeer to be obliged to provide transcoding profiles for the
stream, especially when they might not even know how to configure
those.

When I went through it, I actually went on the dashboard and open
the developer network tab to find out what the dashboard was setting
when creating a stream from the UI, since from there, there is no
requirement for configuring anything.

With this change, it becomes simpler for API users as well, while
still maintaining retro-compatibility and allowing to configure the
profiles manually if desired.

* api/schema: Add default profiles to schema

* api: Add test for default profiles

* api: Fix prettier 😈

gioelecerati committed:
www: added status & status details on webhooks dashboard (#866)

* added status badge on webhooks dashboard

* compare failure and trigger timestamps

* added optional tooltipText

* changed unhealthy logic

* added failure details - fixes

* fix timeframe unhealthy - added status code

* separate rows for messages

gioelecerati committed:
api: fix missing webhook lastFailure (#883)

* fix missing lastFailure

* added unknown error

iameli committed:
optional recaptcha (#884)

* www: don't require recaptcha in dev

* build: add local pkg build

iameli committed:
chore(release): publish v0.1.2

gioelecerati committed:
api: Asset and Task resource types (#865)

* added schema & table for vodAsset and task

* vod nft property to array

* added audio track parameters & better descriptions

* added userId and deleted fields

* remove id and other fields from required on asset

* remove more required

* extended task object

* frameRate to fps

* some changes on examples

* added additionalProperties - task endpoints - hash as object

* added bunch of readOnly

* fixed examples

* fix typo

* changes after review

* changed hash to array, fixed timestamp type

* added parentAssetId and output fields to task object

* enums

* rename vod-asset to asset

* draft task ctrl

* removed task test - added owner to task

* req.store to db - removed try

* js to ts - fixed deleted / fixed db.task.get

* update to patch - only admin

* added asset controller

* added vodObjectStore id param

* added playbackId - objectStoreId writeOnly

* added aws-sdk client and presigner

* added presigned url helper - request-upload and upload proxy asset endpoints

* review changes

* removed check

* added update status endpoint

* added size, pixelFormat, removed trackid

* added proxy callback and task creation

* added tasktrigger and taskresult

* added exchange and queue

* added http-proxy types

* changed queue binding

* added import - changed publish

* taskId in the event key

* added uploadedObjectKey - objectStoreId on asset gen

* edits

* api: Sync asset schema with task runner

* api: Sync a few bits and pieces on assets controller

Mainly:
 - Fixed a couple typescript issues which then showed a few bugs
 - Fix the schema of the task events, now type-checked on callsite
 - Add a couple missing fields in the creation of assets, making sure we
   always have a playbackId in a format consistent with streams'
 - Change a few things on the schema, mainly adding the right status
   enums and making all values lowercase to match the pattern in our
   other "status/type-like" fields (like stream.status itself)

* api/queue: Fix task exchange and queue names

* api: More syncs for tasks to start running!

Mainly:
 - Add object store ID to every asset
 - Update task status API

* api: Remove a couple forgotten logs

* added TaskScheduler

* log

* fixed presignedUrl parsing on upload

* added task update phase status

* output metadata - added indexes

* fix await on result

* api: Fix object-store tests

* lockfile

* prettier

Co-authored-by: Victor Elias <victorgelias@gmail.com>

victorges committed:
chore(release): publish v0.2.0

gioelecerati committed:
api: Add export and transcode to Asset and Task  (#893)

* added export

* added generic handler to handle export

* added scheduleTask - moved pub msgs to scheduleTask

* changed ipfs - removed waiting from outside scheduler

* parentAssetId to inputAssetId - outputAssetId

* typo

* fix playbackId length and prefix

* removed task name - remove originTaskId - changed scheduler to support both output and input assets

* schema: Update export task params+output schema

* api: Fixes to the export asset API

E2E tested, it works!!
https://ipfs.io/ipfs/QmRpyvxmxWPUUNSDXodjqDVutsq1ubH9sc85B57Eo7tB75

* api: Add download URL to assets

* api: Add dynamic IPFS URLs to export task outputs

* added ingest to webhook

* added live-to-vod-asset

* same shardKey as stream - better types on shard schedule

* same shardKey as stream - better types on shard schedule

* api/task: Use our dedicated IPFS gateway

* fixed signed url encoding

* added origin to ingests

* check if asset is ready before export

* check if asset is ready before export

* api/cannon: Remove redundant stream/session fetch

* removed redundant mp4 record url from webhook cannon

* removed baseIngest from cannon

* baseIngest remove

* api/store: Make cleanWriteOnlyResponse recursive

* api: Avoid creating fields that dont exist

* api: Remove transcode task type for now

* api: Fix asset creation on request upload

* api: Clean asset responses as well

* api: Create type for new asset

* api/scheduler: Fix task not getting updated

* api: Do some oneOf magic on export-task-params

* api: Fix asset subfield clean up

* api: Improve task error handling

 - Set asset as failed in case upload fails
 - Do not allow upload if asset has already been uploaded
 - Handle corner cases on task event result procecssing

* api: List assets and tasks by updatedAt desc

* api: Some final nits

* api/task: Set IPFS URLs on regular list as well

* prettier

Co-authored-by: Victor Elias <victorgelias@gmail.com>
Co-authored-by: Victor Elias <victor@livepeer.org>

victorges committed:
chore(release): publish v0.2.1

victorges committed:
api: Fix mp4Url on live2vod

victorges committed:
chore(release): publish v0.2.2

iameli committed:
api: use url parser for getS3PresignedUrl

iameli committed:
chore(release): publish v0.2.3

victorges committed:
api: Fix /asset/import API request payload schema

victorges committed:
chore(release): publish v0.2.4

FullStackTiger committed:
created products/media-server page (#901)

FullStackTiger committed:
Add "Media Server" aka MistServer to footer (#887)

* Add "Media Server" aka MistServer to footer

* updated the link to /products/media-server

FullStackTiger committed:
Update nav "Use Cases" to "Products" and rearrange (#885)

* Update nav "Use Cases" to "Products" and rearrange

* added target attribute

* prettier

* created Media Server icon

* changed the link to /products/media-server

* prettier

FullStackTiger committed:
Add page to Dashboard for "Media Server" under Developers (#886)

* Add page to Dashboard for "Media Server" under Developers

* updated Livepeer Media Server (Mist) to Livepeer Media Server (MistServer)

* prettier

FullStackTiger committed:
added media server docs (#888)

adamsoffer committed:
fix product page build (#909)

Commit:
Adding MistServer copy to documentation (#910)

* Adding MistServer copy to documentation

* prettier

Co-authored-by: FullStackTiger <35409129+FullStackTiger@users.noreply.github.com>

victorges committed:
api: Fix some schemas from asset APIs (#900)

* api/schema: Fix the new-asset-payload schema

* api/schema: Make video and metadata file paths private

* api/asset: Return upload task ID in the reponse headers

* api/schema: Add format validation to import/export URLs

* api: Do Livepeer-Task-Id instead of X-

* api/task: Create task on request-upload API

* api/task/export: Rename ERC1155->NFT Metadata

* api/asset: Do not use replica when fetching asset on upload

Commit:
Livepeer.com homepage Web3 tweaks v01 (#896)

* Updated H2 copy

Note: I'm not sure if the code will react to the &nbsp; I placed in it to ensure there's space after the Livepeer CTA... will test

* Update next-seo.config.js

* Update index.tsx

Part 1 of 2... coming back later for the "Features" section

* Update index.tsx

* Update index.tsx

* Update index.tsx

* Update contact.tsx

* Update index.tsx

* prettier

Co-authored-by: FullStackTiger <35409129+FullStackTiger@users.noreply.github.com>

adamsoffer committed:
return orchestrators pending activation for Confluence upgrade (#911)

victorges committed:
api/task: Add separate IPFS URL as well as gateway's (#914)

* api/task: Add separate IPFS URL as well as gateway's

* Prettify

Commit:
Livepeer.com homepage Web3 tweaks v02 (#913)

* Update index.tsx

* Update index.tsx

* Update contact.tsx

* Update index.tsx

* Update index.tsx

* prettier

Co-authored-by: FullStackTiger <35409129+FullStackTiger@users.noreply.github.com>

FullStackTiger committed:
Add "Downloads" to the MediaServer (#916)

FullStackTiger committed:
updated navigation (#915)

victorges committed:
chore(release): publish v0.2.5

darkdarkdragon committed:
VOD transcode task (#895)

* VOD transcode task

* use only one profile in VOD transcode task

hjpotter92 committed:
ci/release: Use `.zip` for windows release archives. (#897)

Add steps to generate checksum and gpg signature with release artifacts.

hjpotter92 committed:
gitignore: Update patterns for cross-ide development.

 - Drop debug statement from ci release workflow.
 - Add `Makefile` spec to `.editorconfig`.

hjpotter92 committed:
chore(release): publish v0.2.6

hjpotter92 committed:
Makefile: Add Makefile to help with muscle memory when releasing.

FullStackTiger committed:
handling image hero (#919)

Commit:
Update register.tsx (#920)

FullStackTiger committed:
added a section on the logged in /dashboard for MistServer (#921)

gioelecerati committed:
Disable cors on vod api (#922)

* disable cors on vod api

* changed to cors lib

* added method, continue on preflight request, allow credentials

* preflightContinue

FullStackTiger committed:
removed this asterisk on billing and pricing pages (#932)

FullStackTiger committed:
fixed "Unexpected Error" when going to the billing page (#933)

FullStackTiger committed:
Create "verify your email" banner on dashboard home (#931)

* wip

* completed the resend verify email function

* removed unnecessary code

* remove index

hjpotter92 committed:
workflow: Create release only if action runs on a tag. (#934)

FullStackTiger committed:
fixed email verify banner issue (#935)

hjpotter92 committed:
workflow/release: Fix expression syntax error (#936)

hjpotter92 committed:
chore(release): publish v0.2.7

iameli committed:
www: make stripe optional (#948)

* www: optional stripe

* www: cleaner optional stripe

* cleanup from code review

Co-authored-by: Victor Elias <victorgelias@gmail.com>

* add Promise.resolve

Co-authored-by: Victor Elias <victorgelias@gmail.com>

Co-authored-by: Victor Elias <victorgelias@gmail.com>

victorges committed:
api: Rename local database to `livepeer` (as box) (#952)

victorges committed:
www: Page for minting an NFT from an exported asset (#923)

* www: Add transact eth page to make web3 tx

* nit: Message

* www: Fix check for transaction

* Output ID of minted token

* Remove possibility for custom transaction

* Show errors minting NFT

* Remove usage of typechain

* Add a little button to start tx

* Do not be strict on development CLI flags

* Make "Mint NFT" a proper button

* Add timestamp to logs

* Add surrounding dashboard UI

* Make mint NFT a dialog

* Add button to connect to MetaMask

* Switch network if not in Polygon

* Clear logs on connect as well

* nit start msg

* Fix tx result logs

* Add retries to getting minted token ID

* Make it easy to add other networks

* Add arbitrum as a supported network

* Better formatting for account and chain ID logs

* Keep arbitrum off for now

* Fix mint successful log

* www: Rename NFT page to /mint-nft

* www: Add video-nft solidity code backup

* Print tx hash as soon as it's sent

* Disable form fields sometimes

* Do not show placeholder if default contract unavailable

* Add proper gas estimation

* Fix footer position on screen

* Fix page content

* Do not redact hash if no explorer configured

victorges committed:
Fix build

Just making the new option from layout optional

gioelecerati committed:
www: Docs - added video on demand api reference  (#946)

* www: vod - added vod docs

* fix docs positions

* format change

* slight changes after review

adamsoffer committed:
Add docs link to dashboard header (#928)

FullStackTiger committed:
Livepeer.com Docs for MistServer Integration (#959)

* created mistserver docs

* prettier

* added images for the mistserver docs

victorges committed:
www: Disable contract field when not on right network (#960)

* www: Disable contract field when not on right network

It can be confusing since we were showing an empty placeholder
when the network was wrong (no default contract address).

With this, we disable the field and also show an unsupported network
placeholder.

* www: Change placeholder to "Unsupported network"

victorges committed:
www: Add support for polygon testnet on mint-nft (#962)

* www: Add support for polygon testnet on mint-nft

* Fix typo

FullStackTiger committed:
Add Assets Page to Dashboard (#958)

* added assets page to the dashboard

* changed component name

* updated content

FullStackTiger committed:
Redirect MistServer links to /docs (#967)

* updated links and docs

* redirected to /docs/guides/media-server/introduction

FullStackTiger committed:
Updated social media click-thrus in footer (#969)

iameli committed:
api: allow suspending users from admin token (#947)

victorges committed:
api/queue: Fix delayed messages logic (#972)

* api/queue: Revert to old way of delaying messages

* api/queue: Remove old queue deleting logic

* api/queue: Delete previous delayed queues

* api/queue: Create helper to do setup+unsetup

Cleans up the logic a bit

* api/queue: Reduce the number of delayed queues

Removing the routing key from the queue name we
avoid creating so many queues. For that, required
creating a separate exchange for the queue as well
so that the original routing key of the message is
kept in the message metadata, and forwarded to the
deadletter exchange when the message expires.

* api: Add test to catch the old problem

* api/queue: Name queues with the ms value

* api: Add another test to check routing key stays the same

hjpotter92 committed:
api: Update yargs initialisation environment prefix. (#951)

* api: Update yargs initialisation environment prefix.

* api: remove more LP prefixes

Co-authored-by: Eli Mallon <eli@livepeer.org>

FullStackTiger committed:
Added multistream destination bitrate on stream health (#953)

* added multistream destination bitrate on stream health

* handling the Legend fontsize on health chart

* Update stream health page

* fixed coding style

victorges committed:
chore(release): publish v0.3.0

victorges committed:
api: Generic but signed upload urls (#918)

* api/asset: Use signed JWTs as the URLs for validation

* api/asset: Create a couple helpers for gen/parse upload url

victorges committed:
api: Replace web3 with ethers in /mint-nft (#976)

* api: Replace web3 with ethers in /mint-nft

* www/mint-nft: Use transaction receipt to get Mint event

victorges committed:
api: Fix RabbitMQ tests flakiness (#977)

* api: Fix RabbitMQ tests flakiness

We have a lot of tests that fail because one event
leaks from one test to another. With this change, we
create a new vhost in RabbitMQ for every test suite
run, and additionally also create one vhost for every
test run in the specific queue test, which tests the
delayed events and thus could have more leaks.

This will hopefully make them more independent and also
not affect the local development environments (e.g. the box)
whenever we run tests on localhost.

* .github: Run rmq services with management API

* .github: Expose management API port as well

gioelecerati committed:
www: added full mint nft tutorial to docs (#980)

* www: added full mint nft tutoral to docs

* www: DOCS - moved nft docs to guides from api reference

gioelecerati committed:
api: Change user session timeout to 1 min (#979)

* api: changed session timeout to 1 min to shorten rec generation wait time

* api: let non-admin users to use forceUrl in setRecordingStatus

gioelecerati committed:
api: Allow user to retrieve its informations (#956)

* api: allow user to retrieve its informations

* api: changed user tests - removed unverified grants

* api: moved user retrieve logic on a separate /me endpoint

* api: added user test info & allow unverified user

* added check on email field on test

victorges committed:
api: Send user suspension email automatically + suspend all streams (#989)

* api: Create schema for suspend user API

* api: Send account suspension email on user suspended

* bonus: Suspend all user streams on suspension

* api/user: Log with logger instead of console

Didn't even know we had that

* api: Decouple appeal button from unsubscribe

* api: Add bcc to suspension email

* api: Create suspensionEmailText helper

Also always email the user with some simple template,
but use the copyright one if specified.

* Fix tests

victorges committed:
www: Allow sending copyright suspension email from admin dashboard (#990)

* www/admin: Send copyright infringiment email from admin

* www: Fix style

victorges committed:
api/user: bcc the right email (#991)

victorges committed:
chore(release): publish v0.4.0

yondonfu committed:
api: Use lastInitializedRound in subgraph query (#912)

* api: Use lastInitializedRound in subgraph query

* api: Fix lastInitializedRound parsing from subgraph

FullStackTiger committed:
VOD: Implement asset list within Livepeer.com UI (#984)

* wip

* added asset page

* updated the assets table

* updated the filter

victorges committed:
api/stream: Improve reliability of `/setactive` API (#992)

* api/user: Only send email if user not already suspended

* api/stream: Only block request if active=true

We don't want to leave suspended streams indeterminately
active in the database.

* api/stream: Add back /setactive API log

* api/stream: Create schema for /setactive api

* api/stream: Ignore /setactive calls not from last session

* api/stream: Only set user.lastStreamedAt if !!active

* api/stream: Set all /setactive hooks in background

* api/stream: Update only the stream itself in foreground

* Nit a comment

* Who else? Prettier off course

* api: Add log when we ignore set active

victorges committed:
www/mint-nft: Use Video NFT SDK instead of ethers! (#978)

* www/mint-nft: Use Video NFT SDK instead of ethers!

* www/mint-nft: Fix the nft info usage

* Update to latest version of video-nft

* www/mint-nft: Fix switch-network button

* www/mint-nft: Add a file picker

* Update video-nft

* Update sdk version

* Add log for file upload error

* Just use the endpoint from the use-api client

* Remove double /api

* Check existence of window and browser

* mint-nft: Upgrade video-nft SDK version

* mint-nft: Only show Pick file button if no IPFS url set

* mint-nft: Only show upload file button if logged in

* Switch to the final merged version of the SDK!

* www: Use released version of Video NFT SDK

* Fix upload file button

* Update video-nft version

I won't even say it is the last one lol

thomshutt committed:
Add missing word to mistserver features description (#963)

victorges committed:
api: Allow hackers to create dev API keys with CORS (#985)

* api: Split authenticator middleware from the authorizer

* api: Make createdByTokenId an admin-only field

That field basically exposes API keys on all stream
API endpoints. Which means for example that any API
key with access to the /streams API can get access
to any other API key that has ever created streams
(since our API keys are actually the IDs themselves).

This avoids that leak for non-admin users. We should
eventually make the API keys themselves not actually
use the ID as the secret key.

* api: Create separate 401 error different from 403

401 actually means the request is not authenticated (yeah the HTTP name is
    confusing)
403 means that the caller is authenticated but is not allowed to perform
    the action (aka forbidden or unauthorized)

* api: Move CORS and Prometheus to app router

We'll need to add middlewares before CORS now, then
I also didn't want to leave Prometheus all alone in
the root (that test middleware doesnt count).

* api: Validate noApiToken option

* api: Allow API keys to add allowed origins

* api: Add Vary: Authorization to all responses

* api: Fix tests and auth middleware imports

* api: Remove CORS logic from asset controller

* api/cli: Make cors allow-list configured as an arg/env

* api/cors: Make the static allowlist only for JWT reqs

* api/env: Remove localhost from default allowlist

We should still configure prod/staging separately, with
prod only really allowing access from livepeer.com (and maybe explorer?)

* api: Remove explorer from CORS allowlist

Not needed anymore!

Also fix some . that should be \. in the regexes

* api: Default CORS to https://${frontendDomain} instead

* api: Create logic for restricting CORS access

Pending actually allowing some APIs to be used

* api: Allow some stream APIs to use use CORS

* api: Allow some multistream APIs to use CORS

* api: Allow some asset APIs to use CORS

* api: Allow any origin to access /api/asset/upload

* www: Create allow CORS checkbox in API key creation

* www: Add input for allowed origins for API key CORS

* www: Add a + button next to add origin field

* www: Clear API key dialog anytime it is closed

* www: Update copy for less indirection

* www: Allow creating CORS API keys with full API access

* prettier

* www: Fix default suggested CORS origin

* www: Stop disallowing whitespace in token

* www: Remove whitespace from addAllowedOrigin field

* api: Allow any origin in the OPTIONS call

* api: Create getCorsOpts internal helper

Just some cleaner code

* api: Disallow API keys on certain user APIs

Also disallow an user to suspend themelves lol

* api: Fix CORS for API tokens that allow *

* api: Simplify OPTIONS CORS handling

* api: Fix CORS allow-list arg parsing

* api: Make CORS allowed origins list a JSON array

* api/task: Tune authorizer params

* api: Always run cors middleware after auth

* api: Fix auth middleware

express-async-errors is a lie, we can't really
ignore next for a middleware apparently.

* www: Create column for CORS in API key table

* www: Add origin validation

* www: Keep only * when * is in allowed origins

* www/docs: Add first part of API key CORS docs

* www: Add dummy API access docs and links from creation dialog

* api: Change transcode API to /api/asset/:id/transcode

Still keep backward compatibility since the NFT SDK is already
calling the old one.

* www/docs: Fix API key reference

Made it a little shorter just with usage information
and a link to the getting an API key guide.

* www/docs: Remove domain warning from Stream Health

It is not true anymore!

Also remove the repeated beta badge that messes with the
header. We already have the note right below and it's not
**that** beta anymore lol

* www/docs: Finish API key docs!

* www/docs: Final nits!

victorges committed:
www/docs/api-key: Fix some wording

hjpotter92 committed:
workflows: Enable codesign and notarize for macOS binaries (#997)

hjpotter92 committed:
chore(release): publish v0.4.1

gioelecerati committed:
www: Switch cdn.livepeer.com to livepeercdn.com (#1000)

* www: switch cdn.livepeer.com to livepeercdn.com

* www: added double domain check on asset table

* www: docs: rollback regional endpoints

* fix typo

* fix typo

* www: added tooltip on playback URL

iameli committed:
chore(release): publish v0.4.2

hjpotter92 committed:
LICENSE: it's 2022 now :)

hjpotter92 committed:
Upload code coverage reports to codecov (#1006)

victorges committed:
www/docs: Add documentation about NFT and webrtmp SDKs (#1007)

* www/docs: Boilerplate for new Video NFT SDK doc

* www: Require user to be logged in for full /mint-nft

We can't upload a file without an API key.

* www/docs: Improve the mint NFT docs about using the mint-nft page

* www/docs: Add docs for building a Video NFT app

* www/docs: Improve live streaming from your app docs and add browser

victorges committed:
www/docs: Fix typo

FullStackTiger committed:
Temporary Redirect for /catalyst (#1015)

FullStackTiger committed:
Update Contact Us footer copy (#1017)

gioelecerati committed:
schema: add user.new event (#995)

* Added user.new in schema and frontend dropdown

* changed user.new to playback.user.new

iameli committed:
api: no index plz (#1010)

iameli committed:
github actions: fix Docker build globbing (#1030)

* .github: fix action change detection

* packages: no-op change to trigger rebuild

gioelecerati committed:
api: Added /webhook/subscribed/:event api (#994)

* Added user.new in schema and frontend dropdown

* api: added subscribed webhooks api

* api: moved event to params

* userId def

Co-authored-by: Eli Mallon <eli@livepeer.org>

figintern committed:
Tq/task runner prepare (#1022)

* task-runner: prepare task

* Add Playback URL

* Make playbackRecordingId writeOnly and add guard

* withPlaybackUrls refactor

iameli committed:
Update index.tsx

figintern committed:
fix: task-runner asset playbackUrl (#1037)

victorges committed:
api/asset: Return the correct asset on /transcode

We were actually returning the input asset instead,
which made any code expecting the output to have the
ID of the output asset to not work.

This will likely fix https://github.com/livepeer/video-nft/issues/7

FullStackTiger committed:
Update HubSpot form on contact's page (#1043)

* wip with hubspot api

* updated form ui

* updated contact form

* replace endpoint

* Change from Project Aqueduct to Catalyst

adamsoffer committed:
Use latest design-system library (#1032)

adamsoffer committed:
use primary and neutral css vars (#1049)

victorges committed:
api: Do not skip index creation on tests (#1052)

adamsoffer committed:
fix: banner layout (#1055)

thomshutt committed:
Mark half orchestrators in a region as untrusted when flag passed

thomshutt committed:
Mark half orchestrators in a region as untrusted when flag passed

thomshutt committed:
Try passing through param

thomshutt committed:
Try passing through param

thomshutt committed:
Fix variable name

thomshutt committed:
Merge branch 'it/half-untrusted' of github.com:livepeer/livepeer-com into it/half-untrusted

thomshutt committed:
Return async function to allow param to be passed in
